staging/src/k8s.io/apiserver/pkg/admission/plugin/namespace/lifecycle/patch.go

```diff
@@ -0,0 +1,7 @@
+package lifecycle
+
+import "k8s.io/apimachinery/pkg/runtime/schema"
+
+func AccessReviewResources() map[schema.GroupResource]bool {
```

Comment:
as far as I see not used anymore. Please drop.

Reply:
Done.